sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py

```diff
@@ -771,7 +771,7 @@ def _read(self, n, initial=False, buffer=None, _errnos=None):  # pylint: disable
         :return: The data read.
         :rtype: bytearray
         """
-        from websocket import WebSocketTimeoutException
+        from websocket import WebSocketTimeoutException, WebSocketConnectionClosedException
```

Comment:
can we add a test for this or is this only reproduceable manually?

Reply:
manually requires loss